lang/user/id.xml

```diff
@@ -487,7 +474,7 @@ Silakan unduh dan instal.</no_font_exist>
 	</live_area>
 
 	<settings name="Pengaturan">
-		<theme_background name="Tema & Latarbelakang">
+		<theme_background name="Tema Dan Latarbelakang">
```

Comment:
Keep & for this, !is same for all langague

Reply:
OK, Mr. Zangetsu.